### ISSUE TITLE
puzzling annotation 

### ISSUE THREAD

**Issue by z-jiaming** [TRI-ML/VOST#2]
Great work, Congratulations!

But when I was looking at the dataset I noticed a lot of annotations with white parts, I would like to know what these areas mean, thanks!

Example: 1339_cut_aubergine, frame00192

<img width="1146" alt="image" src="https://user-images.githubusercontent.com/42411674/235614757-41e8bfba-0cc9-4a70-bba9-2105f7d9e3a4.png">

**Comment by pvtokmakov** [comment]
Hi Jiaming,

thanks for your interest in our work! These annotations represent "Ignore" regions - areas where it is hard to give a precise label, e.g. due to motion blur, like I this example. They should not be used during training and evaluation.

**Comment by z-jiaming** [comment]
Thanks for your reply!

Could you please add comments to the following part of the code for clearer:
1) the params in L253~L256 in aot_plus/dataloaders/train_datasets.py
2) Where to modify the calculation loss to ignore the "Ignore" regions

Thank you so much!

**Comment by z-jiaming** [comment]
Thanks for your reply a lot!
